arcgis-ios-sdk-samples/Layers/Group layers/GroupLayersSectionView.xib

```diff
@@ -0,0 +1,50 @@
+    <device id="retina4_7" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14460.20"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner" customClass="LayersTableViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="XjP-9q-BD7" customClass="GroupLayersSectionView" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target">
+            <rect key="frame" x="0.0" y="0.0" width="375" height="44"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" horizontalCompressionResistancePriority="749" text="Label" textAlignment="natural" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="10R-ek-huY">
+                    <rect key="frame" x="16" y="5" width="300" height="34"/>
+                    <fontDescription key="fontDescription" type="boldSystem" pointSize="17"/>
+                    <nil key="textColor"/>
+                    <nil key="highlightedColor"/>
+                </label>
+                <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" translatesAutoresizingMaskIntoConstraints="NO" id="9ku-4w-xzb">
+                    <rect key="frame" x="321" y="5" width="51" height="34"/>
+                    <connections>
+                        <action selector="toggleSwitch:" destination="-2" eventType="valueChanged" id="ndi-hb-abW"/>
+                    </connections>
+                </switch>
+            </subviews>
+            <constraints>
+                <constraint firstItem="10R-ek-huY" firstAttribute="top" secondItem="XjP-9q-BD7" secondAttribute="top" constant="5" id="1AJ-iz-MjX"/>
+                <constraint firstItem="9ku-4w-xzb" firstAttribute="leading" secondItem="10R-ek-huY" secondAttribute="trailing" constant="5" id="Lt2-qq-1br"/>
+                <constraint firstItem="10R-ek-huY" firstAttribute="leading" secondItem="XjP-9q-BD7" secondAttribute="leading" constant="16" id="NKS-xX-JVd"/>
```

Comment:
Rather than hardcoding the leading space, please constrain to the superview's leading margin:

![image](https://user-images.githubusercontent.com/2257493/56170831-d9c86480-5f97-11e9-80ce-2850f6985bff.png)

Reply:
If I constrain to the leading space of label and trailing space of switch to super's margin, it appears like this. 
![Screen Shot 2019-04-15 at 5 02 41 PM](https://user-images.githubusercontent.com/8196343/56172903-53fce700-5fa0-11e9-8497-5ca1f2d8864a.png)

Reply:
Did you adjust the spacing constraint between the label and the switch? Currently it is as follows:

![image](https://user-images.githubusercontent.com/2257493/56220138-cdd2b600-601c-11e9-896f-6404367d46df.png)

This means that the trailing edge of the label should be five points after the leading edge of the switch. It translates to the following linear equation: `label.leading = 1.0 * switch.leading + 5.0`. The label is allowed to underlap the switch by five points, which is what you're seeing in that last row.

A better constraint would be `switch.leading >= label.leading + c`, where `c` is some constant, such as the system spacing. It looks as follows in Interface Builder:

![image](https://user-images.githubusercontent.com/2257493/56224862-2e65f100-6025-11e9-9f15-696675f6cddc.png)

In order for that last row to be sized (and truncated) properly, you'll need to set the number of lines to `1`:

![image](https://user-images.githubusercontent.com/2257493/56225177-b8ae5500-6025-11e9-8db2-859ce4a7769c.png)

![Screen Shot 2019-04-16 at 8 59 55 AM](https://user-images.githubusercontent.com/2257493/56225819-eb0c8200-6026-11e9-97ac-a504026051fa.png)

Alternatively, you could change the relation of the switch's top and bottom constraints to `greaterThanOrEqual` and then center it vertically, allowing the label to grow to two lines, thereby increasing the height of that cell.

![image](https://user-images.githubusercontent.com/2257493/56225662-aaad0400-6026-11e9-97b3-2aa6eef709eb.png)